redis-lettuce/build.gradle

```diff
@@ -1,10 +1,15 @@
+plugins {
+    id "io.micronaut.build.internal.module"
+}
+
 dependencies {
     annotationProcessor "io.micronaut:micronaut-inject-java"
-    // needed because otherwise IntelliJ integration broken
-    compileOnly "io.micronaut:micronaut-inject-java"
     annotationProcessor "io.micronaut.docs:micronaut-docs-asciidoc-config-props:$micronautDocsVersion"
     annotationProcessor "io.micronaut:micronaut-graal"
 
+    // needed because otherwise IntelliJ integration broken
+    compileOnly "io.micronaut:micronaut-inject-java"
```

Comment:
that looks weird. Can you verify we need this nowadays?

Reply:
Doesn't look like we do.  At least I couldn't see any issues without it.

Fixed in https://github.com/micronaut-projects/micronaut-redis/pull/257/commits/fac6b96acdf69aeaf71c137967b87de20e524a8a